CHANGELOG.unreleased.md

```diff
@@ -27,6 +27,7 @@
 - [Jobs] Fix "invalid parameters" error when running jobs in some cases (PR [#6156](https://github.com/vatesfr/xen-orchestra/pull/6156))
 - [New SR] Take NFS version and options into account when creating an ISO SR
 - Allow a decimal when displaying small values (e.g. show *1.4 TiB* instead of *1 TiB* for 1,400 GiB of RAM)
+- [vmdk] Fix `VBOX_E_FILE_ERROR (0x80BB0004)` when trying to manipulate a vmdk exported from xo in virtual box (PR [#6163](https://github.com/vatesfr/xen-orchestra/pull/6163))
 
 ### Packages to release
```

Comment:
Rebase on `master` and add all the packages that should be released in order for this fix to reach users :slightly_smiling_face: